src/main/webapp/i18n/de/editor.json

```diff
@@ -85,7 +85,8 @@
                 "problemStatementCouldNotBeUpdated": "Die Aufgabenstellung der Übung konnte nicht aktualisiert werden.",
                 "failedToLoadBuildLogs": "Die Build Logs konnten nicht geladen werden.",
                 "repositoryInConflict": "Dein Repository befindet sich in einem Konfliktzustand.",
-                "notAllowedExam": "Du kannst (nicht mehr) abgeben"
+                "notAllowedExam": "Du kannst (nicht mehr) abgeben",
+                "connectionTimeout": "Zeitlimit überschritten. Speichern ist fehlgeschlagen."
```

Comment:
it's not really clear what you mean with "Zeitlimit" this is too technical. I would rather translate it to user language.